{
  "name": "@progresskinvey/kinvey-datalink-errors",
  "version": "0.3.4",
  "description": "Error mapping for Datalink Connectors",
  "license": "Apache-2.0",
  "repository": {
    "type": "git",
    "url": "Kinvey/kinvey-datalink-errors"
  },
  "engines": {
    "node": "= 0.10.x"
  },
  "devDependencies": {
    "eslint": "8.15.0",
    "eslint-config-kinvey-platform": "0.1.6",
    "eslint-plugin-import": "2.26.0",
    "mocha": "10.0.0",
    "should": "13.2.3"
  },
  "scripts": {
    "pretest": "npm run lint",
    "lint": "./node_modules/.bin/eslint lib test *.js --color",
    "test": "npm run test-unit && npm run test-integration",
    "test-unit": "mocha test/unit/*/*",
    "test-integration": "echo \"No integration tests available for the moment\"",
    "test-npm-security": "npm audit --production --audit-level=high"
  },
  "dependencies": {
    "fix": "0.0.6"
  }
}
